src/tests/ftest/util/dmg_utils.py

```diff
@@ -355,7 +355,10 @@ def get_pool_uuid_from_stdout(stdout_str):
     # Find the following with regex. One or more of whitespace after "UUID:"
     # followed by one of more of number, alphabets, or -. Use parenthesis to
     # get the returned value.
-    matches = re.findall(r"UUID:\s+([0-9a-fA-F-]+)", stdout_str)
-    if len(matches) > 0:
-        return matches[0]
-    return None
+    uuid = None
+    svc = None
+    match = re.search(r" UUID: (.+), Service replicas: (.+)", stdout_str)
```

Comment:
I think it would be better if the method name is updated to something like get_uuid_service_repolica_from_stdout